lib/chewy/search/response.rb

```diff
@@ -19,6 +19,12 @@ def hits
         @hits ||= hits_root['hits'] || []
       end
 
+      # Raw response body obtained from ES.
+      # @param [Hash, nil]
```

Comment:
Param or return? (also would be nice as sig :) )

Reply:
Yes return 🤦‍♂️